server.js

```diff
@@ -1,4 +1,5 @@
 require('dotenv').config();
+require('./config/mogoose');
```

Comment:
Please split database config and database connection

Reply:
I created a database folder and moved mongoose.js to it